e2e/provision/README.md

```diff
@@ -267,20 +318,30 @@ machinesets`. You should see READY and AVAILABLE replicas.
 kubectl get machinesets
 ```
 
-The output is similar to:
-```
+<details>
+<summary>The output is similar to:</summary>
+
+```console
 NAME                                   CLUSTER    REPLICAS   READY   AVAILABLE   AGE     VERSION
 regional-md-0-zhw2j-58d497c498xkz96z   regional   3          3       3           3h58m   v1.26.3
```

Comment:
Need to change the value to one as we change the kind cluster deployment with 1 worker.